### DISCUSSION TITLE
Firefly error log

### DISCUSSION THREAD

**Discussion by kongmingzhgl** [iotaledger/firefly#8644] [Help & Support]
Receiving the error log when retrieving old wallet after updating the software:
Generic: {"type":"error","payload":{"type":"client","error":"node error: error sending request for url (https://iota-node.tanglebay.com/api/core/v2/info): operation timed out"}}

Please advise how can I restore the old wallet. Thanks.

**Comment by begonaalvarezd** [comment]
Do you have an active firewall or use a VPN? looks like your machine is blocking the node
